Makefile

```diff
@@ -1205,5 +1253,4 @@ lint-clean:
   test-v8-benchmarks \
   test-v8-intl \
   uninstall \
-  v8 \
-  website-upload
+  v8
```

Comment:
Should this have been a separate commit?  It looks like an unrelated change.

Reply:
@bnoordhuis I did this when documenting the `-upload` targets and this one does not seem valid any more. I could separate this one out in another commit (there are actually quite a few invalid phony targets)